proposals/description/fip_60.ts

```diff
@@ -0,0 +1,153 @@
+import { ProposalDescription } from '@custom-types/types';
+
+const fip_60: ProposalDescription = {
+  title: 'FIP-60: FeiRari Rewards Upgrade',
+  commands: [
+    // TODO add TC Sync actions before and after
```

Comment:
finish todo item

Reply:
think we shouldn't do this in the OA proposal actually. @ElliotFriedman wdyt